Legible/Classes/MatchingSnapshot.swift

```diff
@@ -3,6 +3,7 @@ import Nimble
 import SwiftUI
 
 public class SnapshotConfiguration {
+    static public var usedSnapshots = [URL]()
```

Comment:
seems like `usedSnapshots` do not belong to configuration, may be move to something like `SnapshotResults`